root/static/scripts/work.js

```diff
@@ -295,7 +294,7 @@ function renderWorkLanguages() {
       removeClassName="remove-language"
       removeLabel={l('Remove Language')}
       repeatable={form.field.languages}
-      noLyrics={form.field.languages.errors}
+      noLyrics={_.intersection(form.field.languages.field.map(lang => String(lang.value)), ["486", "284"]).length > 0}
```

Comment:
* I guess `String(lang.value)` was just a test _to be sure_. Since it is known to be `number`, it would be simpler to just compare it with number.
* This function call returns a boolean, but you changed the type of `noLyrics`.
* `noLyrics` should rather be named something general-purpose like `hideAddButton`.

Reply:
Okay, will make these changes.
However, it does seem like lang.value is stored as a string.
When I originally load the page, the value is stored as an int when viewed in the React chrome extension, but after editing the value, it's turned into a string. So it seems like it needs to be converted at some stage?

Reply:
Oh right, sorry, keep it as string then.